UnityProject/Assets/Scripts/Items/Devices/AccessRemote.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using Doors;
+using Doors.Modules;
+using Mirror;
+using UnityEngine;
+
+
+namespace Items.Devices
+{
+	/// <summary>
+	/// An item that controls department doors remotely.
+	/// </summary>
+	public class AccessRemote : NetworkBehaviour, ICheckedInteractable<HandActivate>, ICheckedInteractable<HandApply>
+	{
+		[SyncVar] private AccessRemoteState currentState;
```

Comment:
```suggestion
		private AccessRemoteState currentState;
```

This doesnt need to be a sync var as the client isnt using it anywhere.